manim/mobject/text/tex_mobject.py

```diff
@@ -416,17 +429,19 @@ def set_color_by_tex_to_color_map(self, texs_to_color_map, **kwargs):
                     self.set_color_by_tex(tex, color, **kwargs)
         return self
 
-    def index_of_part(self, part):
+    def index_of_part(self, part: MathTex) -> int:
```

Comment:
Could `part` be a `SingleStringMathTex`, since those are the submobjects of `MathTex`?
```suggestion
    def index_of_part(self, part: SingleStringMathTex) -> int:
```